src/main/java/uk/gov/cslearning/record/service/DefaultBookingService.java

```diff
@@ -91,6 +90,12 @@ public BookingDto unregister(Booking booking) {
         return unregister(bookingDtoFactory.create(booking));
     }
 
+    @Override
+    public Optional<Booking> isLearnerBookedOnEvent(String learnerEmail, String eventUid){;
```

Comment:
By convention, methods names that start with 'is' return booleans. This should probably be renamed to findActiveBookingByEmailAndEvent (or something).